### ISSUE TITLE
Enabling vim extension in vscode disables my keyboard to type anything in the editor. 

### ISSUE THREAD

**Issue by prateekgurnani10** [VSCodeVim/Vim#4400]
I cannot type anything inside the editor. I can delete though. My keyboard just doesn't work for typing. I disabled the vim extension and then I am able to type. 
- Extension (VsCodeVim) version: 1.12.2
- VSCode version: 1.41
- OS:  Mac os Catalina 10.15.2

**Comment by J-Fields** [comment]
Have you tried re-installing?

**Comment by prateekgurnani10** [comment]
Yes, but still it doesn't work. it only works when i have disabled the extension.

**Comment by montogeek** [comment]
Same issue, It is there a way to debug it?
I activated the `Window log`, and nothings pops out there.

**Comment by KrishKashiwala** [comment]
I am also same kind of error. Any updates how to solve it?

**Comment by Sugoi-TI** [comment]
Same here guys. Used it previously and everything was fine, except for some strange bugs.

1 disabled it
2 installed nvim plugin
3 nvim works even worse
4 uninstall nvim
5 enable vim plugin and now it does this keyboard block

**Comment by Sugoi-TI** [comment]
for me removing  this line from settings.json do the trick "vim.vimrc.enable": true

**Comment by DragonGong** [comment]
It might be that the settings.json under .vscode has affected the global setting.json. Deleting the settings.json under .vscode and restarting worked for me.
